src/schema/document/value.rs

```diff
@@ -146,16 +145,16 @@ pub enum ReferenceValueLeaf<'a> {
     F64(f64),
     /// Date/time with nanoseconds precision
     Date(DateTime),
-    /// Facet
-    Facet(&'a Facet),
+    /// Facet, needs to match the format of [Facet].
```

Comment:
I think this should clarify that the _encoded_ format is meant, not the human-readable slash-separated one.